.github/workflows/node.js.yml

```diff
@@ -10,28 +10,45 @@ on:
         branches: [master, feature/*]
 
 jobs:
-    build:
-        name: Build
+    macos:
+        name: macOS nodejs
         runs-on: macos-latest
-
         strategy:
             matrix:
-                node-version: [10.x]
-
+                node-version: [12.x]
         steps:
             - uses: actions/checkout@v2
             - name: Use Node.js ${{ matrix.node-version }}
-              uses: actions/setup-node@v1
+              uses: actions/setup-node@v2
               with:
                   node-version: ${{ matrix.node-version }}
             - run: npm install
             - run: npm run vscode:prepublish
               env:
                   NODE_OPTIONS: '--max-old-space-size=8192'
-            - name: Run headless test
-              uses: GabrielBB/xvfb-action@v1.2
+            - name: Tests
+              uses: GabrielBB/xvfb-action@v1
```

Comment:
Why the version downgrade?

Reply:
Not intended to be a downgrade. The current version is 1.4. I think specifying "1" will use the latest 1.x version. This avoids needing to manually bump the minor versions.

Reply:
Sounds good, I didn't realize it would work in a "1.x" capacity.

Reply:
It's not documented AFAICT, but implied by this thread: https://github.community/t/referencing-major-version-of-github-action-only-fails-to-download-action/17129/3

which hints that it is convention-based and depends on how actions authors manage their repo tags. In the case of xvfb, comparing [v1](https://github.com/GabrielBB/xvfb-action/tree/v1) to [v1.4](https://github.com/GabrielBB/xvfb-action/tree/v1.4) shows that v1 has the latest 1.x (currently 1.4). And the author also [updated their readme](https://github.com/GabrielBB/xvfb-action/commit/e65d7e3bbc395498be1d95c40b38248489e39676) for this purpose.

I will add this info to the commit message.